_episodes/04-data-types-and-format.md

```diff
@@ -15,7 +15,12 @@ objectives:
     - "Analyze datasets having missing/null values (NaN values)."
     - "Write manipulated data to a file."
 keypoints:
-    - "FIXME"
+    - Python uses two main data types text and numeric, these have a different names in basic Python and Pandas
+    - Each column in a DataFrame has a shared data type for all rows
+    - Any data type can be converted to another type
```

Comment:
1. Data types can not be converted. Variables can.
2. Not any data type can be converted to another type (DataFrame to boolean).
Why do you think this keypoint is important?

Reply:
right!, the point is about checking types with dtype, please see changes